### ISSUE TITLE
Suggestion: Cache popular addons

### ISSUE THREAD

**Issue by trlkly** [Noitidart/Chrome-Store-Foxified#38]
I thought it might be a good idea to cache signed addons (that work properly in Firefox), so that you don't keep on signing the same extensions over and over. I realize that space and bandwidth would be limited, so my suggestion is to only have the most popular addons.  

It would also help out those people who say they can't get an addon signed. They could just download a version that has already been signed from a cached server. 

This would also allow for easy automatic updates, if a newer version gets signed.

**Comment by Noitidart** [comment]
Thanks for this idea trlkly. I used to do this in the original design, I had a single AMO account that everyone would use, so things would get cached there. However it was against AMO policies so I had to change to an account per person.

If an addon can't get signed, no one can get it signed, its always due to an issue with the AMO validation process and the progress of the WebExtensions API. :(

But maybe I can add into foxifed "Did this extension work for you?" which will post to the github wiki of foxified and it will add that link in or something.

**Comment by trlkly** [comment]
Right, you can't host it on AMO. But I was thinking you maybe could host
them on your own server (maybe even on Github), and offer to let people
install those. As long as the addons are signed, you can host them
anywhere.

I do like the idea of a "Did this extension work for you?" question, but I
would suggest displaying the results on the actual Chrome store page.
Something like "X users say this works on Firefox 50" or such.
